gulpfile.js

```diff
@@ -22,27 +22,29 @@
  * SOFTWARE.
  */
 
+var del = require('del');
+
 var gulp = require('gulp');
-var clean = require('gulp-clean');
 var concat = require('gulp-concat');
 var server = require('gulp-express');
 var jshint = require('gulp-jshint');
+var less = require('gulp-less');
 var rename = require('gulp-rename');
 var strip = require('gulp-strip-comments');
+var taskListing = require('gulp-task-listing');
 var uglify = require('gulp-uglify');
 var wrap = require('gulp-wrap');
-var taskListing = require('gulp-task-listing');
```

Comment:
Oups, it seems a mistake ?
=> With this plugin, you can type gulp help and it display list of available tasks, useful, isn't it ?